packages/components/src/EmptyState/EmptyState.module.scss

```diff
@@ -7,19 +7,22 @@
   --empty-state-border-width: var(--border-width-1);
 
   display: flex;
-  justify-content: space-around;
   width: 100%;
+  box-sizing: border-box;
+  justify-content: space-around;
   padding: calc(2% - var(--empty-state-border-width))
     calc(var(--spacing-24) - var(--empty-state-border-width));
   color: $color-purple-800;
   border: var(--empty-state-border-width) solid var(--empty-state-border-color);
   border-radius: var(--border-solid-border-radius);
   background-color: var(--empty-state-background-color);
+  overflow: hidden;
```

Comment:
We'd prefer to avoid this because it might end up truncating content unintentionally, do we definitely need it?

Reply:
Removed it!